scripts/docker-compose/check-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
```

Comment:
Add some doc about what this script does? Not immediately clear to me. Maybe also rename it a bit to make it more clear?

Reply:
Renamed to check-for-running-containers.sh